bin/cleanup

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Why not set `-eu` here?

Reply:
It is inherited and controlled from the caller (`test-` files).

Reply:
It is inherited and controlled from the caller (`test-` files).